hack/validate-lint.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-
-errors=$($GOBIN/golangci-lint run --build-tags "${BUILDTAGS}" 2>&1)
-
-if [ "$?" -eq 0 ]; then
-	echo 'Congratulations!  All Go source files have been linted.'
-else
-	{
-		echo "Errors from golangci-lint:"
-		echo "$errors"
-		echo
-		echo 'Please fix the above errors. You can test via "golangci-lint" and commit the result.'
```

Comment:
\*shrug\* IIRC things like this were originally added because external contributors were confused about some of the CI failures. But that must have happened with some earlier linter, and it’s not strong reason to keep this.

Reply:
Yeah, I think that golangci-lint-action is verbose these days, clearly showing the errors in the annotated diffs.